pkgs/tools/admin/certbot/default.nix

```diff
@@ -35,6 +34,7 @@ python37Packages.buildPythonApplication rec {
   ];
 
   buildInputs = [ dialog ] ++ (with python37Packages; [ mock gnureadline ]);
+  nativeBuildInputs = [ python37Packages.setuptoolsCheckHook ];
```

Comment:
as long as `format = "setuptools";` is passed to `buildPython*` (its the default), this hook is included.